modules/kml/docs/api-reference/gpx-loader.md

```diff
@@ -0,0 +1,34 @@
+# GPXLoader
+
+The `GPXLoader` parses [GPX files][gpx_wikipedia] into GeoJSON.
+
+| Loader                | Characteristic                             |
+| --------------------- | ------------------------------------------ |
+| File Extension        | `.gpx`                                     |
+| File Type             | Text                                       |
+| File Format           | [GPX][gpx_wikipedia]                       |
+| Data Format           | [GIS](docs/specifications/category-gis.md) |
+| Decoder Type          | Synchronous                                |
+| Worker Thread Support | No                                         |
+| Streaming Support     | No                                         |
+
+[gpx_wikipedia]: https://en.wikipedia.org/wiki/GPS_Exchange_Format
+
+## Usage
```

Comment:
A bit more information about what is in these files would be super helpful, so that I can evaluate what this format brings at a glance. I think that is a valid comment for all the loaders, but we can start adding this here and now.